diff --git a/browser/components/preferences/preferences.xhtml b/browser/components/preferences/preferences.xhtml
index 53bdc9c912a6cd0c787dbcf41f2a82b7169d5aa7..3f494dc4cdc9850fb1cd8c6ac3b0ae8dbfd17b76 100644
--- a/browser/components/preferences/preferences.xhtml
+++ b/browser/components/preferences/preferences.xhtml
@@ -166,16 +166,6 @@
           <image class="category-icon"/>
           <label class="category-name" flex="1" data-l10n-id="settings-pane-labs-title"></label>
         </richlistitem>
-        <richlistitem id="category-more-from-mozilla"
-                      class="category"
-                      hidden="true"
-                      data-l10n-id="more-from-moz-category"
-                      data-l10n-attrs="tooltiptext"
-                      value="paneMoreFromMozilla"
-                      align="center">
-          <image class="category-icon"/>
-          <label class="category-name" flex="1" data-l10n-id="more-from-moz-title"></label>
-        </richlistitem>
 
         <!-- The following <richlistitem> elements are hidden from the list of items via the hidden-category class.
             The intent is that they act as sub-pages, accessible from the UI of one of the main pages, but are
