diff --git a/browser/app/profile/firefox.js b/browser/app/profile/firefox.js
index 8cbe147d48e858bb8069e4627b3b3903cf8325c0..4e5613c4518f274604511fc1b4cbf28b6dfc920f 100644
--- a/browser/app/profile/firefox.js
+++ b/browser/app/profile/firefox.js
@@ -2583,13 +2583,20 @@ pref("browser.migrate.interactions.passwords", false);
 
 pref("browser.migrate.preferences-entrypoint.enabled", true);
 
-pref("extensions.pocket.api", "api.getpocket.com");
-pref("extensions.pocket.bffApi", "firefox-api-proxy.cdn.mozilla.net");
-pref("extensions.pocket.bffRecentSaves", true);
-pref("extensions.pocket.enabled", true);
-pref("extensions.pocket.oAuthConsumerKey", "40249-e88c401e1b1f2242d9e441c4");
-pref("extensions.pocket.oAuthConsumerKeyBff", "94110-6d5ff7a89d72c869766af0e0");
-pref("extensions.pocket.site", "getpocket.com");
+// pref("extensions.pocket.api", "api.getpocket.com");
+pref("extensions.pocket.api", "");
+// pref("extensions.pocket.bffApi", "firefox-api-proxy.cdn.mozilla.net");
+// pref("extensions.pocket.bffRecentSaves", true);
+pref("extensions.pocket.enabled", false);
+// pref("extensions.pocket.oAuthConsumerKey", "40249-e88c401e1b1f2242d9e441c4");
+pref("extensions.pocket.oAuthConsumerKey", "");
+// pref("extensions.pocket.oAuthConsumerKeyBff", "94110-6d5ff7a89d72c869766af0e0");
+// pref("extensions.pocket.site", "getpocket.com");
+pref("extensions.pocket.site", "");
+user_pref(
+  'browser.newtabpage.activity-stream.section.highlights.includePocket',
+  false
+)
 
 // Enable Pocket button home panel for non link pages.
 pref("extensions.pocket.showHome", true);
@@ -3338,3 +3345,5 @@ pref("toolkit.contentRelevancy.enabled", false);
 pref("toolkit.contentRelevancy.ingestEnabled", false);
 // Pref to enable extra logging for the content relevancy feature
 pref("toolkit.contentRelevancy.log", false);
+
+#include silica.js
