diff --git a/browser/components/customizableui/CustomizableUI.sys.mjs b/browser/components/customizableui/CustomizableUI.sys.mjs
index e6b9b7dcfab179e7552c146eb1551b45ad042266..80ebffbb27309baeb454d6a1fbb6ff284c61b2de 100644
--- a/browser/components/customizableui/CustomizableUI.sys.mjs
+++ b/browser/components/customizableui/CustomizableUI.sys.mjs
@@ -317,7 +317,7 @@ var CustomizableUIInternal = {
       "vertical-spacer",
       "urlbar-container",
       "spring",
-      "save-to-pocket-button",
+      // "save-to-pocket-button",
       "downloads-button",
       AppConstants.MOZ_DEV_EDITION ? "developer-button" : null,
       "fxa-toolbar-menu-button",
@@ -331,8 +331,8 @@ var CustomizableUIInternal = {
         overflowable: true,
         defaultPlacements: navbarPlacements,
         verticalTabsDefaultPlacements: [
-          "firefox-view-button",
-          "alltabs-button",
+          // "firefox-view-button",
+          // "alltabs-button",
         ],
         defaultCollapsed: false,
       },
@@ -356,10 +356,10 @@ var CustomizableUIInternal = {
       {
         type: CustomizableUI.TYPE_TOOLBAR,
         defaultPlacements: [
-          "firefox-view-button",
+          // "firefox-view-button",
           "tabbrowser-tabs",
           "new-tab-button",
-          "alltabs-button",
+          // "alltabs-button",
         ],
         verticalTabsDefaultPlacements: [],
         defaultCollapsed: null,
